crates/rune/src/doc/visitor.rs

```diff
@@ -4,6 +4,7 @@ use crate::compile::{
 };
 use crate::hash::Hash;
 
+#[derive(Clone)] // TODO(TSol): this is a bit hackish
```

Comment:
This was necessary for the docs lookup; it didn't seem to work right when putting unit and the associated docs on the workspace. Didn't investigate too much.